pixi.toml

```diff
@@ -144,15 +147,28 @@ pyqt-stubs = "*"
 types-requests = "*"
 qgis-plugin-manager = "*"
 libgdal-arrow-parquet = "3.7.2.*"
+juliaup = "*"
```

Comment:
We typically like to sort our dependencies.
With vscode you can do that by select the lines,  pressing `ctrl-shift-p` and then select `sort lines ascending`